packages/suite-native/src/services/metadata/DropboxProvider.ts

```diff
@@ -16,12 +16,16 @@ class DropboxProvider implements AbstractMetadataProvider {
         return true;
     }
 
+    // @ts-ignore
     async getCredentials() {}
 
+    // @ts-ignore
     async getFileContent() {}
+
+    // @ts-ignore
```

Comment:
Why are these being ignored?

Reply:
they extend abstract class which requires all children to implement methods that are marked as `abstract` on its interface.